packages/core/src/sass/app/modules/account-types.scss

```diff
@@ -0,0 +1,113 @@
+@import '../_common/base/index';
```

Comment:
Keep an eye out on this, since this soon will be moved to deriv/shared package.

Reply:
@reza-fs @mahdiaryayi-fs safe to merge, it won't be moved to shared, it will be only declared in Core. Refer: https://github.com/binary-com/deriv-app/pull/933